neurots/extract_input/from_neurom.py

```diff
@@ -99,13 +102,83 @@ def soma_data(pop):
     return {"size": transform_distr(ss)}
 
 
-def trunk_neurite(pop, neurite_type=nm.BASAL_DENDRITE, bins=30):
+def trunk_vectors(morph, neurite_type):
+    """This is `neurom.get('trunk_vectors')` but wrt to `[0, 0, 0]`."""
+    return [
+        nm.morphmath.vector(n.root_node.points[0], [0.0, 0.0, 0.0])
```

Comment:
indeed, but if one provides non-centered morphologies to synthesis, it will give very weird angles haha. But it's hard to have a warning to detect that. We could see if somebody bumps into this issue badly at some point

Reply:
I am not sure I understand why it would give weird angles if the soma center is always taken into account and the generation of points is done with respect to that.

Reply:
it's for consistency, as we assign trunk angles first, then draw a soma, which center of mass will not be our original [0, 0, 0] position from which we've drawn angles, thus for example validation plots will have differences.

Reply:
Still, imo the reconstruction's center should be checked and raise an error if it's not centered at the origin. There is no guarantee that users will pass centered morphologies to the tool.

Reply:
this is no more